landbosse/excelio/XlsxValidator.py

```diff
@@ -45,6 +45,25 @@ def compare_expected_to_actual(self, expected_xlsx, actual_module_type_operation
             'USD/kW per project': 'usd_per_kw_per_project'
         }, inplace=True)
 
+        ## Suggested changes... needs some additional development
```

Comment:
This tests for something different than the original code implementation. This implementation is slightly more permissive than the original (since it can still pass with different row orders) but that is a minor edge case. But this way has fewer lines of code to maintain. I'll go ahead and finish this implementation, and we can go with that.